CHANGELOG.md

```diff
@@ -11,8 +11,9 @@
   - `sequence`
 - Added support for registering and calling Stored Procedures with `TABLE` return type.
 - Added support for parameter `length` in `StringType()` to specify the maximum number of characters that can be stored by the column.
-- Added the alias `functions.element_at()` for `functions.get()`
-- Set default reviwer
+- Added the alias `functions.element_at()` for `functions.get()`.
+- Set default reviewer.
```

Comment:
I know this is not part of your change, just wanna note that we should remove this before releasing.

Reply:
I was also of the same opinion. I'll keep in mind the make sure this is removed.